src/node_crypto.cc

```diff
@@ -741,6 +743,56 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void SecureContext::UseExtraCaCerts(const char* fname) {
+  extra_root_certs_file = fname;
+}
+
+static void emitWarning(Environment* env, const char* fmt, ...) {
+  char warning[1024];
+  va_list ap;
+
+  va_start(ap, fmt);
+  vsnprintf(warning, sizeof(warning), fmt, ap);
+  va_end(ap);
+
+  Local<Object> process = env->process_object();
+  Local<v8::Value> f = process->Get(
+    FIXED_ONE_BYTE_STRING(env->isolate(), "emitWarning"));
+  Local<Value> args[] = {
+    node::OneByteString(env->isolate(), warning, strlen(warning))
+  };
+
+  f.As<v8::Function>()->Call(process, arraysize(args), args);
```

Comment:
@trevnorris I think that MakeCallback can cause running microtasks as a side-effect (https://github.com/nodejs/node/blob/09d41e22cc9bd3c35b454dc239c616be12e26a73/src/node.cc#L1268), which would not be desireable, but I'm not sure how to demonstrate this. Trevor, you know this code, should usage like above use direct `Call()`, or `node::MakeCallback()`?

My failed attempt at making a "microtask" :-( at https://gist.github.com/927a451526de55847d53694b53b01a20